python/ray/worker.py

```diff
@@ -958,9 +958,13 @@ def cleanup(worker=global_worker):
                               {"end_time": time.time()})
     services.cleanup()
   else:
-    # If this is not a driver, make sure there are no orphan processes.
+    # If this is not a driver, make sure there are no orphan processes, besides
+    # possibly the worker itself.
```

Comment:
This is mostly a sanity check that a worker process didn't start any other processes.